src/Mantasflowers.Domain/Entities/User.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using Mantasflowers.Domain.Enums;
+
+namespace Mantasflowers.Domain.Entities
+{
+    public class User : BaseEntity
+    {
+        public string FirstName { get; set; }
+
+        public string LastName { get; set; }
+
+        public bool IsRegistered { get; set; }
```

Comment:
```cs
public bool IsRegistered { get; set; }
```
Seems to be redundant. Is there a use case for this bool?

Reply:
We'll have to allow customers to buy without registering. I was initially thinking that we could create them as User entities nonetheless (because they still might need some JWT based auth to get their in-progress order). These users could then be without contacts or address (those would get filled in at the moment of making purchase).

Reply:
One thing to consider, how are we going to prevent some other customer from being associated with the stored unregistered user?

Reply:
Don't know. Maybe we could issue JWT's to such customers our selves. But then we would need to store them, which kinda defeats the purpose :(

Reply:
Maybe backend can be oblivious to such customers until the very end? I.e. only frontend knows about them?

Reply:
That being the case, we wouldn't have to periodically prune unregistered users from database, however, contact and address separation is fine, in case we decide to change/extend some information fields. Main concern would be if we really want to make orders traceable to unregistered users.